en/dev_setup/dev_env_windows_vm.md

```diff
@@ -44,3 +44,24 @@ VMWare performance is acceptable for basic usage (building Firmware) but not for
     * Clipboard sharing to host system
     * File sharing to host system
 1. Continue with [PX4 environment setup for Linux](../dev_setup/dev_env_linux.md)
+
+## VirtualBox 7 USB passthrough for QGroundControl / Firmware Flashing
+
+In order to flash newly-built firmware onto autopilot hardware, there's a need to configure USB passthrough settings.
+
+:::tip
+This guide is for VirtualBox 7 users hosting Ubuntu 20.04 LTS on a Windows 10 machine.
+:::
+
+1. Ensure that the user has been added to the dialout group using `sudo usermod -a -G dialout $USER` and restart
+1. Enable serial port(s) in VM: **VirtualBox > Settings > Serial Ports 1/2/3/etc...**
+1. Enable USB controller in VM: **VirtualBox > Settings > USB**
+1. Add USB filters for the bootloader and autopilot in VM: **VirtualBox > Settings > USB > Add new USB filter**
+   ... Autopilot: This shows up at a device after the bootloader finishes the booting process
+   ... Bootloader: This shows up as a device for a few seconds after plugging in the USB cable, you only have a few seconds to add to filter before it disappears as a device.
```

Comment:
Hi @Ionstream

THanks very much. I've done minor subedit for style - e.g. you can autonumber lists using `1.`, which makes moving things around easier.

So to be clear, what you're doing is building PX4 in a VirtualBox VM and you want to set up a connection so that you can:
- do `update` at the end of your `make px4_watever upload` and upload to a Pixhawk connected to the host computer
- ? Connect to QGC running on the computer ? (not sure)

Is that right?

I'm a bit confused about these two lines - I think it is saying that you select those settings and a dialog will show you some filters you might add. If you plug in a Pixhawk you might see the Bootloader first, and you should select that quickly, then you will see the Autopilot, and that shows up while you're connected after booting? 
I think you're also saying that the text will look like "Something `Autopilot`" (i.e. ... Autopilot) for the usb filter. Is that all right?

Reply:
Hi Hamish,
Apologies for the confusion, I think you got the gist of it.

There's 2 parts to consider:
1. In Windows Device Manager, PX4 hardware shows up as 2 entities - (1) the bootloader for FC (2) the actual FC. However, bootloader and FC will never show up together at the same time; once the booting process is completed for the FC, the bootloader entity disappears and the FC entity shows up in Windows Device Manager.
2. The filters are VM settings for devices that are passed through. Unfortunately, in order to add a filter for (1) and (2), they need to be seen in Windows Device Manager. While not a problem for (2), it is a problem for (1) since the bootloader for the FC only persists for a few seconds in Windows Device Manager.

Reply:
Thanks very much.